superset-frontend/src/components/ReportModal/index.tsx

```diff
@@ -181,11 +193,10 @@ const ReportModal: FunctionComponent<ReportProps> = ({
   const [currentReport, setCurrentReport] = useReducer<
     Reducer<Partial<ReportObject> | null, ReportActionType>
   >(reportReducer, null);
-  const onChange = useCallback((type: any, payload: any) => {
+  const onReducerChange = useCallback((type: any, payload: any) => {
```

Comment:
curious why the switch from onChange to onReducerChange. If I'm reading this right, when the reducer changes, you update the reducer?

Reply:
In cases like [this one](https://github.com/preset-io/superset/blob/5812a79c0e8eb6cf70297de197534ea9e62a4b52/superset-frontend/src/components/ReportModal/index.tsx#L294-L299), there was originally an `onChange` nested into another `onChange`. I thought that since there were two `onChange` functions in the component it could get confusing, so I renamed the reducer's `onChange` to `onReducerChange` to clarify that `onReducerChange` is specifically the function that the reducer uses to change state. Is that an unnecessary clarification, or is there maybe a better name I can use here?

Reply:
Ok, I gotcha. Normally the naming convention will be the element that is changing, rather than the function that is used to do the change, thus the confusion. Maybe something like `onInputChange` would make sense.

Reply:
Good call, `onInputChange` sounds a lot better! I'll change the naming once I reopen this to fix it.